### PR TITLE
Update microsoft-defender-endpoint-mac.md

### PR DESCRIPTION
**Pull request by msbemba**
For now M1 is still not supported- latest blog -https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/microsoft-defender-atp-for-mac-is-moving-to-system-extensions/ba-p/1608736

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4709

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-mac.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ The three most recent major releases of macOS are supported.
 
 Beta versions of macOS are not supported.
 
+MAC Systems with M1 processors are not supported.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-mac.md]:
```suggestion
macOS devices with M1 processors are not supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestion only.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**